backend/src/main/java/de/neuefische/backend/controller/RandMController.java

```diff
@@ -26,4 +29,15 @@ public List<RandMCharacter> fillCharactersFromApi(){
     }
 
 
+    @GetMapping("/game/board/generate")
+    public List<RandMCharacter> generateBoardByCondition(@RequestParam(required = false) Integer m,
+                                                         @RequestParam(required = false) Integer n,
+                                                         @RequestParam(required = false) BoardGenerationCondition condition) {
+
+        return randMService.generateBoardByCondition(
+                Objects.requireNonNullElse(m, 4),
```

Comment:
common use variables should be x & y, as usual in a coordinate system. It wasn't immediately clear what the variables m & n should stand for

Reply:
The choice of "m" and "n" as variable names originates from mathematics, where these letters are frequently used to represent generic variables in equations and formulas.
"m" and "n" represent the dimensions or sizes of the game board. So I think it's a matter of taste.
Ideally, there should be documentation of the API, which, unfortunately, we do not have yet.

Reply:
"you never stop learning" :D